apps/web/content/docs/en/core/accounts.mdx

```diff
@@ -237,18 +238,18 @@ Account {
 }
 ```
 
-## !!steps Owner Field
+## !!steps Owner field
 
-The program ID (public key) of the program that owns this account.
+The `owner` field lists the program ID (public key) of the program that owns this account.
 
-Every Solana account has a designated program as its owner. Only the program
-that owns an account can change the account's data or deduct its lamports
-balance.
+Every Solana account has a designated [program](/docs/core/programs) as its owner. Only the program
+that owns an account can change the account's `data` field or deduct lamports from its balance.
 
-The instructions defined in a program determine how the account's data and
-lamports balance can be changed.
+For executable accounts, the `owner` field contains the program ID of a loader
+program. [Loader programs](https://solana.com/docs/core/programs#loader-programs) are built-in programs responsible for loading and
```

Comment:
These docs have a LOT of explanatory asides. I think we should remove them for the most part. In this case, let's move the [Loader programs](https://solana.com/docs/core/programs#loader-programs) link to the previous sentence and get rid of the explanation